RELEASE_NOTES.md

```diff
@@ -76,7 +76,7 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
 - Add new raster mask supports. This is a copy of a parametric mask
   which is stable during the whole pixel-pipe.
 
-- Modules can now be re-ordered with <kbd>shift+drag</kbd>.
+- Modules can now be re-ordered with <kbd>Ctrl+shift+drag</kbd>.
```

Comment:
I suppose we want a capital S in Shift at this stage (after all standardization work on key shortcuts).